bin/reth/src/node/mod.rs

```diff
@@ -540,6 +549,27 @@ impl<Ext: RethCliExt> NodeCommand<Ext> {
 
         self.ext.on_node_started(&components)?;
 
+        // If `enable_genesis_walkback` is set to true, the rollup client will need to
```

Comment:
can we link to somewhere where this behavior of the op-node is explained? and let's use consistent terminology if "rollup client" refers to op-node or equivalent